packages/contentful--app-scripts/README.md

```diff
@@ -106,14 +106,15 @@ When passing the `--ci` argument the command will fail when the required variabl
 
 **Options:**
 
-| Argument            | Description                                                                                                                                |
-| ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------ |
-| `--bundle-dir`      | The directory of your build folder (e.g.: `./build`)                                                                                       |
-| `--organization-id` | The ID of your organisation                                                                                                                |
-| `--definition-id`   | The ID of the app to which to add the bundle                                                                                               |
-| `--token`           | A personal [access token](https://www.contentful.com/developers/docs/references/content-management-api/#/reference/personal-access-tokens) |
-| `--skip-activation` | (optional) Boolean flag to skip the automatic activation of the `AppBundle`                                                                |
-| `--comment`         | (optional) A comment which will be associated with the created `AppBundle`. Can be used to differentiate bundles.                          |
+| Argument            | Description                                                                                                                                | Default value |
+| ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------ | - |
+| `--bundle-dir`      | The directory of your build folder (e.g.: `./build`)                                                                                       | |
+| `--organization-id` | The ID of your organisation                                                                                                                | |
+| `--definition-id`   | The ID of the app to which to add the bundle                                                                                               | |
+| `--token`           | A personal [access token](https://www.contentful.com/developers/docs/references/content-management-api/#/reference/personal-access-tokens) | |
+| `--skip-activation` | (optional) Boolean flag to skip the automatic activation of the `AppBundle`                                                                | `false` |
+| `--comment`         | (optional) A comment which will be associated with the created `AppBundle`. Can be used to differentiate bundles.                          | |
+| `--host`            | (optional) Contentful CMA-endpoint to use                                                                                                  | `upload.contentful.com` |
```

Comment:
Really? I think it's `api.contentful.com` and we infer the upload endpoint here: https://github.com/contentful/create-contentful-app/blob/59f76a04fcd5929511161d770e1a949e4e7c0019/packages/contentful--app-scripts/lib/upload/create-app-upload.js#L8